### PR TITLE
mysql++ 3.2.3

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I'm not saying nobody cares about this formula but version `3.2.2` was released at the start of 2015 & nobody noticed 😅.

### CODE REVIEW

**Comment by ilovezfs** [comment]
142 unique install since 1st Jan

**Comment by DomT4** [comment]
I guess if it isn't broken it might as well stick around, but yeah, colour me unsurprised that number is low.

File: Formula/mysql++.rb [hunk 1]
```diff
@@ -15,9 +15,8 @@ class Mysqlxx < Formula
   depends_on :mysql
 
   def install
-    mysql_include_dir = `mysql_config --variable=pkgincludedir`
```
Comment (ilovezfs) [inline Formula/mysql++.rb]: flare up of Domitus?

Comment (DomT4) [inline Formula/mysql++.rb]: Can't deal with backticks in formulae 🙈

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
:snail:

**Pull request closed without merging**